go.mod

```diff
@@ -1,5 +1,42 @@
 module mosaicmfg.com/stl-to-3mf
 
-go 1.16
+go 1.22
+
+toolchain go1.23.1
```

Comment:
Should this version not match the Go version? i.e. should they both be 1.22 or 1.23.1?